packages/manager/modules/pci/src/components/project/storages/databases/text-accordion/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import '@ovh-ux/ui-kit';
+
+import directive from './directive';
+import controller from './controller';
+
+import './styles.scss';
+
+const moduleName = 'ovhManagerPciComponentsTextAccordion';
+
+angular
+  .module(moduleName, ['oui'])
+  .controller('textAccordionController', controller)
+  .directive('textAccordion', directive);
```

Comment:
When not expanded, maybe you can add `text-overflow: ellipsis` to add the 3 dots indicating that the text is truncated.
I have tried with a very long request, and it does seems to limit to two lines, with the first one truncated.

### Not expanded:
![image](https://user-images.githubusercontent.com/24541481/146024984-74193aff-1da6-41e6-8364-36157cc43ac3.png)

### Expanded
![image](https://user-images.githubusercontent.com/24541481/146025055-50058929-d88c-44f1-8189-db7e5bd96cd0.png)

### The (very) long request
![image](https://user-images.githubusercontent.com/24541481/146025193-7304971f-0776-49c5-b0e9-0dc7aab3be5c.png)